source/includes/_forecast_processes.md

```diff
@@ -17,31 +28,31 @@ __role__ | `operator`, `supervisor`
     "status_code": 200,
     "payload": [
         {
-            "id": "aggregate_process_1",
-            "name": "Aggregate process 1",
+            "id": "main_process_1",
```

Comment:
I don't think these should be labelled processes, they're forecast types (e.g. "steam", "electricity")